WordPress/Tests/Theme/NoFaviconUnitTest.inc

```diff
@@ -0,0 +1,6 @@
+/* ----- Bad ------- ERROR ---- */
+<link rel="shortcut icon" href="<?php myfunction('link' ); ?>">
+<link rel="icon" href="<?php myfunction('link' ); ?>">
+<link rel="apple-touch-icon" href="<?php myfunction('link' ); ?>">
+<link rel="apple-touch-icon-precomposed" href="<?php myfunction('link' ); ?>">
+<meta name="msapplication-TileImage" href="<?php myfunction('link' ); ?>">
```

Comment:
Please add some valid meta tags and link tags to make sure the sniff doesn't trigger on those.